_es/tour/multiple-parameter-lists.md

```diff
@@ -10,12 +10,35 @@ next-page: automatic-closures
 previous-page: nested-functions
 ---
 
-_Nota de traducción: Currying es una técnica de programación funcional nombrada en honor al matemático y lógico Haskell Curry. Es por eso que no se intentará hacer ninguna traducción sobre el término Currying. Entiendase este como una acción, técnica base de PF. Como una nota al paso, el lenguaje de programación Haskell debe su nombre a este eximio matemático._
+_Nota de traducción: Currying es una técnica de programación funcional nombrada en honor al matemático y lógico Haskell Curry. Es por eso que no se intentará hacer ninguna traducción sobre el término Currying. Entiéndase este como una acción, técnica base de PF. Como una nota al paso, el lenguaje de programación Haskell debe su nombre a este eximio matemático._
 
 Los métodos pueden definir múltiples listas de parámetros. Cuando un método es invocado con un número menor de listas de parámetros, en su lugar se devolverá una función que toma las listas faltantes como sus argumentos.
 
-Aquí se muestra un ejemplo:
+### Ejemplos
 
+A continuación hay un ejemplo, tal y como se define en el trait `TraversableOnce` en el API de collecciones de Scala:
```

Comment:
Mispring: Collections in Spanish is "colecciones" instead of "co**ll**ecciones".